doc/FEATURES_RELEASE_2.md

```diff
@@ -61,10 +60,10 @@ Enter the root of the repository. Run the command
 $ ./export-mock-ddi-data.sh harmful harmful.csv
 ```
 
-The file `harmful.csv` will be constructed, which can be viewed in a text editor.
-The output of this command is also readily available to view in the
-[mock-enriched-ddis directory](../mock-enriched-ddis/harmful.csv) of our repository.
-
+The file `harmful.csv` will be constructed, which can be viewed in a text
+editor. The output of this command is also readily available to view in
+the [mock-enriched-ddis directory](../mock-enriched-ddis/harmful.csv) of our
+repository.
```

Comment:
Do you think we should mention that this data is the _exact_ data that the system uses in production? Just to be very clear. (same thing for the other mock data)